crates/biome_css_formatter/tests/specs/scss/at-rule/each.scss

```
@@ -45,6 +45,38 @@ puma,sea-slug,egret,salamander{
 
 @each $element,$size in(h1:20px,h2:16px,h3:14px){}
 
+@each $element,$size in (h1:20px,h2:16px,h3:14px){}
```

Stylelint (17.11.1)

[error] Empty block (block-no-empty)

(block-no-empty)